Ginger/GingerCoreCommon/ReporterLib/UserMsgsPool.cs

```diff
@@ -821,6 +822,7 @@ public static void LoadUserMsgsPool()
 
             Reporter.UserMsgsPool.Add(eUserMsgKey.UpdateExistingPOMElement, new UserMsg(eUserMsgType.QUESTION, "Updated Element Found", "An updated version of Page Object Model Element '{0}' Found." + Environment.NewLine + "Do you want to update existing POM ?", eUserMsgOption.YesNo, eUserMsgSelection.Yes));
             Reporter.UserMsgsPool.Add(eUserMsgKey.SavePOMChanges, new UserMsg(eUserMsgType.QUESTION, "Save POM Changes", "Selected POM '{0}' was updated." + Environment.NewLine + "Do you want to save changes ?", eUserMsgOption.YesNo, eUserMsgSelection.Yes));
+            Reporter.UserMsgsPool.Add(eUserMsgKey.ShareApplicationToEnvironment, new UserMsg(eUserMsgType.QUESTION, "Add Application All Environment", "Do you want to publish this application to all the existing environments" , eUserMsgOption.YesNo, eUserMsgSelection.None));
```

Comment:
change it to "Add Application To All Environment", To is missing

Reply:
Implemented this